apps/wallet-mobile/src/components/ConfirmTxWithSpendingPasswordModal/ConfirmTxWithSpendingPasswordModal.tsx

```diff
@@ -0,0 +1,120 @@
+import React, {useRef} from 'react'
+import {ActivityIndicator, StyleSheet, TextInput as RNTextInput, View} from 'react-native'
+
+import {debugWalletInfo, features} from '../../features'
+import {useSelectedWallet} from '../../SelectedWallet'
+import {COLORS} from '../../theme'
+import {WrongPassword} from '../../yoroi-wallets/cardano/errors'
+import {useSignTxWithPassword, useSubmitTx} from '../../yoroi-wallets/hooks'
+import {YoroiSignedTx, YoroiUnsignedTx} from '../../yoroi-wallets/types'
+import {Button} from '../Button'
+import {Spacer} from '../Spacer'
+import {Text} from '../Text'
+import {TextInput} from '../TextInput'
+
+type Props = {
+  onSuccess?: (signedTx: YoroiSignedTx) => void
+  unsignedTx: YoroiUnsignedTx
+  strings: {
```

Comment:
@MichalSzorad  Are strings passed as props because of the modal not working with hooks thing?